src/Host/Broker/Impl/Sessions/SessionsController.cs

```diff
@@ -54,15 +56,20 @@ public class SessionsController : Controller {
                 interp = _interpManager.Interpreters.First();
             }
 
-            var session = _sessionManager.CreateSession(User.Identity, id, interp, securePassword, profilePath, request.CommandLineArguments);
-            return Task.FromResult<IActionResult>(new ObjectResult(session.Info));
+            try {
+                var session = _sessionManager.CreateSession(User.Identity, id, interp, securePassword, profilePath, request.CommandLineArguments);
+                return Task.FromResult<IActionResult>(new ObjectResult(session.Info));
+            } catch (Exception ex) {
```

Comment:
Basically this all mostly to give user something they either understand or at least report to use when logs are not available and then are unable to send feedback.